Revit_Core_Engine/Create/Definition/ProjectParameter.cs

```diff
@@ -42,10 +42,10 @@ public static partial class Create
         [Input("instance", "If true, the created parameter will be an instance parameter, otherwise it will be a type parameter.")]
         [Input("categories", "Categories, to which the created parameter is bound. It will get bound to all categories if this value is null.")]
         [Output("definition", "Revit project parameter Definition created based on the input properties.")]
-#if (REVIT2020 || REVIT2021 || REVIT2022)
+#if REVIT2020 || REVIT2021  || REVIT2022
         public static Definition ProjectParameter(Document document, string parameterName, ParameterType parameterType, BuiltInParameterGroup parameterGroup, bool instance, IEnumerable<Category> categories)
 #else
-        public static Definition ProjectParameter(Document document, string parameterName, ForgeTypeId parameterType, BuiltInParameterGroup parameterGroup, bool instance, IEnumerable<Category> categories)
```

Comment:
Same as previously, I would rather create another if/else for 2025 to avoid changing the original behaviour...